src/server/pfs/server/paths.go

```diff
@@ -59,6 +60,14 @@ func cleanPath(p string) string {
 	return "/" + strings.Trim(p, "/")
 }
 
+// adds a trailing slash to the path if not already present
```

Comment:
Does this make `pachctl copy file repo1@master repo2@master` the same as `pachctl copy file repo1@master:/ repo2@master:/`?